docs/modules/ROOT/pages/vshn-managed/how-tos/custom-image-keycloak.adoc

```diff
@@ -0,0 +1,32 @@
+= How to create a customization container image for Keycloak
+
+This page describes how to create a container image with custom `themes` and `providers` to use along with `VSHNKeycloak` service.
```

Comment:
```suggestion
This page describes how to create a container image with custom https://www.keycloak.org/docs/latest/server_development/#_themes[themes] and https://www.keycloak.org/docs/latest/server_development/#_providers[providers] to use along with `VSHNKeycloak` service.
```

Reply:
When you add a ^ before the closing square bracket, e.g.: https://www.keycloak.org/docs/latest/server_development/#_themes[themes^] the link will open in a new tab. I usually always do that for external links.

Reply:
that's really cool feature from Antora, thanks